Dockerfile

```diff
@@ -23,33 +23,32 @@
 #   docker run -ti --rm -e HOST_IP=$(ip route | grep -v docker | awk '{if(NF==11) print $9}') --entrypoint /bin/bash local/pfcon
 #
 
-FROM fnndsc/ubuntu-python3:latest
-MAINTAINER fnndsc "dev@babymri.org"
+FROM python:3.8.6-buster AS build
```

Comment:
There is no longer a need for `AS build`, see https://github.com/FNNDSC/pfcon/pull/76/files#r550116964